README.md

```diff
@@ -86,11 +87,13 @@ environment variable
 
 ##### Steps for Windows platform
 
-* Go to directory with cloned repository
+* Go to the directory with the cloned repository and enter the subfolder tools\ms.
```

Comment:
@anosach-luxoft fixed in [07ed25a](https://github.com/smartdevicelink/sdl_core/pull/791/commits/07ed25aea7676d504be507c2c11b640656576e3e)